server/lib/routes/get-fxa-client-configuration.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+module.exports = function (config) {
+  var route = {};
+  route.method = 'get';
+  route.path = '/.well-known/fxa-client-configuration';
+
+  var fxaClientConfig = {
+    /*eslint-disable camelcase */
+    auth_server_base_url: config.get('fxaccount_url'),
+    oauth_server_base_url: config.get('oauth_url'),
+    profile_server_base_url: config.get('profile_url'),
+    sync_tokenserver_base_url: config.get('sync_tokenserver_url'),
+  };
+
+  var maxAge = config.get('fxa_client_configuration.max_age');
+  if (maxAge) {
```

Comment:
This precludes the option of setting the max-age to 0 to disable caching. Is that the intent? FWIW, I kinda feel like we should have a cache-control header builder somewhere. I think we have similar code elsewhere.